src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
Thanks for the clarification.

So this NewRelic API doesn't have a notion of logging anything besides an error? The idea here is that this appender intentionally only logs errors?

> (NewRelic/noticeError ?err params)

Is there a reason you don't want more of the `data` content for this call?

All you're be logging in this case is the exception and level. Not the namespace, logging message, args, etc.

Any reason you wouldn't want, for example, to add `(output-fn data)` to the `params` map here?